velox/functions/prestosql/Arithmetic.h

```diff
@@ -495,5 +499,23 @@ struct TruncateFunction {
   }
 };
 
+template <typename T>
+struct BetaCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double a, double b, double value) {
+    VELOX_USER_CHECK((a > 0) && (b > 0), "alpha and beta must be > 0");
```

Comment:
Let's split this check into two separate checks and use VELOX_USER_CHECK_GT for better error messages.

Let's also match Presto's error messages:

```
presto:di> select beta_cdf(-1, -1, 0);
Query 20230426_132309_08035_wz4f2 failed: a must be > 0

presto:di> select beta_cdf(0.5, -1, 0);
Query 20230426_132317_08038_wz4f2 failed: b must be > 0
```

Reply:
Fixed it